### ISSUE TITLE
OpenAPI 3.0 version format

### ISSUE THREAD

**Issue by stojsavljevic** [AtomLinter/linter-swagger#79]
Hi,

thanks for your great project.

I just noticed that OpenAPI specs need to have `openapi` version in quotes for `linter-swagger` to kick in.
Example:
```
openapi: 3.0.0
info: ...
```
doesn't trigger linter but:
```
openapi: "3.0.0"
info: ...
```
does.
As far as I know, version doesn't have to be in quotes when using yaml.

**Comment by Arcanemagus** [comment]
Should be as simple as making the `['"]` optional here if you want to put up a PR 😉:
https://github.com/AtomLinter/linter-swagger/blob/2c3a0093a12cde71255a457ac6f06d80c7d47feb/lib/main.js#L12
